TeachingRecordSystem/src/TeachingRecordSystem.SupportUi/Pages/Persons/PersonDetail/EditInduction/CheckYourAnswers.cshtml

```diff
@@ -21,7 +21,9 @@
                         <govuk-summary-list-row-key>Induction status</govuk-summary-list-row-key>
                         <govuk-summary-list-row-value>@Model.InductionStatus</govuk-summary-list-row-value>
                         <govuk-summary-list-row-actions>
-                            <govuk-summary-list-row-action href="@LinkGenerator.InductionEditStatus(Model.PersonId, Model.JourneyInstance!.InstanceId, fromCheckAnswers: JourneyFromCheckYourAnswersPage.CheckYourAnswers)" visually-hidden-text="new start date">Change</govuk-summary-list-row-action>
+                            <govuk-summary-list-row-action
+                                href="@LinkGenerator.InductionEditStatus(Model.PersonId, Model.JourneyInstance!.InstanceId, fromCheckAnswers: JourneyFromCheckYourAnswersPage.CheckYourAnswers)"
+                                visually-hidden-text="new induction status">Change</govuk-summary-list-row-action>
```

Comment:
Unrelated to these changes but should these `visually-hidden-text` attributes have the 'new' prefix? I would think 'Change induction status' reads better than 'Change new induction status'.

Reply:
Makes sense to me - I was trying to interpret what would be wanted from an equivalent alerts CYA page and obvs missed a nuance in the reasoning of what it should be. I have some changes to do on CYA formatting so I'll add to that